save-frontend/src/main/kotlin/com/saveourtool/save/frontend/App.kt

```diff
@@ -303,6 +313,21 @@ class App : ComponentWithScope<PropsWithChildren, AppState>() {
     }
 }
 
+/**
+ * The function creates routers with the given [href] and ending of string with all the elements given Enum<T>
+ *
+ * @param href
+ * @param routeElement
+ */
+inline fun <reified T : Enum<T>>ChildrenBuilder.createRoutersWithPathAndEachListItem(href: String, routeElement: ReactNode?) {
+    enumValues<T>().map { it.name.lowercase() }.forEach { item ->
+        Route {
+            path = "$href/$item"
+            element = routeElement
+        }
+    }
+}
```

Comment:
```suggestion
inline fun <reified T : Enum<T>>ChildrenBuilder.createRoutersWithPathAndEachListItem(basePath: String, routeElement: ElementType) {
    enumValues<T>().map { it.name.lowercase() }.forEach { item ->
        Route {
            path = "$href/$item"
            element = routeElement.create()
        }
    }
}
```
because you call `create` at every invocation

Also, `href` usually refers to the whole URL, while you are using path without a host here